test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -4537,6 +4538,54 @@ TEST_P(ExtProcIntegrationTest, RequestResponseAttributes) {
 
   verifyDownstreamResponse(*response, 200);
 }
+
+TEST_P(ExtProcIntegrationTest, MappedAttributeBuilder) {
```

Comment:
Thanks for adding the integration test.  Could you please add another integration test to cover per-route-override case?